### DISCUSSION TITLE
lncRNA and microRNA analysis

### DISCUSSION THREAD

**Discussion by najibveto** [biostars/biostar-handbook-issues#204] [Ideas]
hello,
I would like to suggest if it is possible to add a chapter about analysis of non-coding RNA such lncRNA and microRNA.
it will be interesting to learn about it.
thank you.

**Comment by ialbert** [comment]
Thank you for the suggestion, will be considered. 

By the way, I have recently become aware of a fairly extensive review on  the subject (though I have a not studied it myself):

Genome-Wide Analysis of Human Long Noncoding RNAs: A Provocative Review

https://www.annualreviews.org/doi/abs/10.1146/annurev-genom-112921-123710

**Reply by najibveto** [reply]
thanks a lot for the book, i learnt a lot from it especially transcriptome analysis. the most difficult part for me is working with non model organisms (fish) which doesn't have a well annotated genome.
